debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -269,6 +270,16 @@ public static DecimalHandlingMode parse(String value, String defaultValue) {
             .withDescription("A comma-separated list of regular expressions matching fully-qualified names of columns that should "
                     + "be masked by hashing the input. Using the specified hash algorithms and salt.");
 
+    public static final Field MASK_COLUMN = Field.create("column.mask.with.(d+).chars")
```

Comment:
Have you checked whether/how that placeholder with d+ appraoch works with Kafka Connect? I.e. is there perhaps a warning about an unknown field being given in the logs? But then perhaps that was the case already before?

Reply:
I looked at the Kafka and the connector log. 
Only the connector log contains information.
But it's only the normal startup parameter info form the connector configuration.

So I don't think there is a warning about an unknown field. But maybe I have only looked into the wrong log file.

Reply:
Ok, thanks!